template/www/cordova.js

```diff
@@ -2064,7 +2064,7 @@
         // Revert title bg color
         function revertTitleBarColor () {
             var appView = Windows.UI.ViewManagement.ApplicationView.getForCurrentView();
```

Comment:
Ditto (move into if block?)

Reply:
I think that makes sense in both cases but it was unrelated to the PR.

Reply:
@IndieSW I guess you are right. I think it should have been done in the change PR #232 (279816743f995510a1070a0bf77a3180f11e468a). I think it would be better not to execute `Windows.UI.ViewManagement.ApplicationView.getForCurrentView()` if it is not needed. I guess we should raise another JIRA issue to fix this one?

Reply:
Yes please. 
Unless @IndieSW just wants to piggyback that change onto their PR... that is going to be merged... soon...

Reply:
I agree that the calls are unnecessary outside of the if blocks but I'm not really sure about the Cordova procedures.  Raising another issue would likely get the job done.

Reply:
Ok. I think I can merge this then.

Reply:
I just raised CB-13802 (https://issues.apache.org/jira/browse/CB-13802) to handle the issues with the unneeded calls (along with a couple "else crash on 8.1" comments).

Thanks @janpio for merging this one.